### ISSUE TITLE
transparent vae input range

### ISSUE THREAD

**Issue by mingkimon-dev** [RedAIGC/Flux-version-LayerDiffuse#17]
hi all, 
thank you for sharing the work. im trying to enclude transparent vae into my model but faced numerical problem when calculate the input padded_img_rgb.

transparent vae requires 3 inputs:
1. img_rgba
2. img_rgb
3. padded_img_rgb, which gain by using pad_rgb just like what you did in demo_i2i.py

now im kinda of confused by the output range, cuz when calculating the padded_img_rgb using pad_rgb i gain a huge range number

`target_rgba_bchw = target_rgba.detach().cpu().float() # the target_rgba is a tensor range in -1 to 1
target_rgba_bhwc_list = [x.movedim(0, -1).numpy() for x in target_rgba_bchw]
padded_img_rgb_np = np.stack([pad_rgb(x) for x in target_rgba_bhwc_list], axis=0)  # (B, H, W, 3)`

and furthermore i want to check the other 2 inputs' range too
img_rgba: both rgb and alpha part in range [-1, 1]
img_rgb: in range [-1, 1]

does anyone know what the expected value range of padded_img_rgb should be after pad_rgb? and am i right about the ranges for img_rgba/img_rgb? lots of thanxxxxxxxxxxxx

**Comment by mingkimon-dev** [comment]
i guess it should be in range [-1,1] too? from transparent decode function, i saw the padded_img_rgb need to cat with img_rgba's rgb part(which is in range [-1, 1]

<img width="816" height="238" alt="Image" src="https://github.com/user-attachments/assets/982ccd92-c4a0-441b-a008-94f4b6564756" />
